pkgs/development/go-modules/generic/default.nix

```diff
@@ -257,8 +255,5 @@ let
   });
 in if disabled then
   throw "${package.name} not supported for go ${go.meta.branch}"
-else if modSha256 != null then
-  lib.warn "modSha256 is deprecated and will be removed in the next release (20.09), use vendorSha256 instead" (
```

Comment:
Can you convert this in a throw statement that provides instruction to use `vendorSha256` instead?

Reply:
Changed to throw.